django_loose_fk/fields.py

```diff
@@ -201,6 +201,16 @@ def __get__(self, instance, cls=None):
         if instance is None:
             return self
 
+        # for chained loose-fk fields:
+        if isinstance(instance, ProxyMixin):
+            fk_value = instance._loose_fk_data.get(self.fk_field_name, None)
+            if fk_value:
+                return fk_value
+
+            url_value = instance._loose_fk_data.get(self.fk_field_name, None)
+            if url_value:
+                return url_value
```

Comment:
shouldn't this check `self.url_field_name`?

Also, the return type of the `__get__` is different if you return the URL (`str`) - I'd expect this entire block to just not be there and fall into the `remote_loader.load(...)` path instead, or am I reading things wrong?

Reply:
You are right, I've removed this part, it was unnecessary and incorrect